src/torcontrol.cpp

```diff
@@ -608,7 +608,11 @@ void TorController::disconnected_cb(TorControlConnection& conn)
     service = CService();
     if (!reconnect)
         return;
-    LogPrintf("tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+
+    if (LogAcceptCategory("tor")) {
+        LogPrintf("tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+    }
```

Comment:
If you just change to LogPrint? ( https://github.com/laanwj/bitcoin/blob/2015_11_development_guidelines/doc/developer-notes.md#strings-and-formatting )